tests/foreman/maintain/test_maintenance_mode.py

```diff
@@ -82,6 +88,13 @@ def _finalize():
     assert result.status == 1
     assert 'Maintenance mode is Off' in result.stdout
 
+    if local_advisor_enabled:
+        result = sat_maintain.execute('systemctl status iop-service-vuln-vmaas-sync.timer')
+        assert result.status == 0
+        assert 'Active: active (running)' in result.stdout
+        result = sat_maintain.execute('systemctl list-timers')
+        assert 'iop-service-vuln-vmaas-sync.service' in result.stdout
```

Comment:
What is this asserting? The fact that sync.timer really triggers sync.service (if active)?